### PR TITLE
Template documentation focus

### PR DESCRIPTION
**Pull request by hokei**
[Project Reno Jira Ticket](http://jira.nypl.org/browse/RENO-XXX)

[ResearchNow Jira Ticket](http://jira.nypl.org/browse/SFR-XXX)

## **This PR does the following:**
- Adds template stories and styles

### Front End Review:
- [ ] View [the example in Storybook](https://reno-XXX-nypl.pantheonsite.io/themes/custom/nypl_emulsify/pattern-lab/public)
- [ ] Check against the [Metronome documentation](http://themetronome.co/components/xxx/?fresh=true)

### CODE REVIEW

File: src/react-components/icons/svg/arrow-xsmall.svg [hunk 1]
```diff
@@ -1 +1,3 @@
-<svg width="14" height="9" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M5.984 7.578L.424 1.514 1.9.163l5.22 5.695L12.341.163l1.475 1.351-5.56 6.064a1.542 1.542 0 01-2.272 0zm1.474-1.351z"/></svg>
+<svg width="8" height="12" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (helenvholmes) [inline src/react-components/icons/svg/arrow-xsmall.svg]: Can we actually revert this for the time being? I think I'd like to do icons all at once. (Unless this is just a save-as with a code formatter?)

File: src/react-components/src/components/Template/Template.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+        <div className="content-top"><Placeholder>Content Top</Placeholder></div>
+
+
+        <div className={bem("content-primary", ["with-sidebar-right"])}>
+
+        <Placeholder>Main Content</Placeholder>
+        </div>
+
+        <div className={bem("content-secondary", ["with-sidebar-right"])}><Placeholder>Left Sidebar</Placeholder></div>
```
Comment (helenvholmes) [inline src/react-components/src/components/Template/Template.stories.tsx]: Placeholder content should be "Right Sidebar".

File: src/react-components/src/components/Template/Template.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    <div className={bem("layout-container")}>
+      <header className={bem("header")}><Placeholder>Header</Placeholder></header>
+
+      <main className={"main"}>
+        <div className="content-header">{<Placeholder>Hero</Placeholder>}</div>
+
+        <div className="content-top">{<Placeholder>Content Top</Placeholder>}</div>
+
+        <div className={"content-primary"}>
+        <Placeholder>Main Content</Placeholder>
```
Comment (helenvholmes) [inline src/react-components/src/components/Template/Template.stories.tsx]: A few places with funky formatting on L19 + L25... they should be indented

**Review by helenvholmes — CHANGES_REQUESTED** [review]
Some v v tiny nits that are holdovers from the last PR, but this is looking good!

File: src/react-components/icons/svg/arrow-xsmall.svg (hunk 1 above)
Comment (hokei) [reply src/react-components/icons/svg/arrow-xsmall.svg]: yeah I'll revert it

**Review by helenvholmes — APPROVED** [review]

**Pull request merged**